Sephora-M/assignment3/LetterGridTest.java

```diff
@@ -0,0 +1,50 @@
+package assignment3;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * Note: should improve test by using setup and by testing edge cases...
+ * @author Sephora-M
+ *
+ */
+class LetterGridTest {
+	
+	@Test
+	void testInDictionnary() {
+		String[] words = {"CAR","CARDS","CART","CAT"};
+		Trie dictionnary = new Trie(words);
+		
+		for(String word : words)
+			assertTrue(dictionnary.isWord(word));
+	}
+	
+	@Test
+	void testNotInDictionnary() {
+		String[] words = {"CAR","CARDS","CART","CAT"};
+		Trie dictionnary = new Trie(words);
+
+		assertFalse(dictionnary.isWord("CATS"));
+		assertFalse(dictionnary.isWord(""));
+	}
+	
+	@Test
```

Comment:
Maybe a few more edge-cases if you have the time: empty grid, empty dictionary, words that would work if re-using letters was allowed, words that would work if skipping cells was allowed, words on diagonals etc.